src/synthesizer/python/mod.rs

```diff
@@ -416,9 +416,9 @@ impl Reference {
             Origin::PseudoParameter(x) => match x {
                 PseudoParameter::Partition => "self.partition".into(),
                 PseudoParameter::Region => "self.region".into(),
-                PseudoParameter::StackId => "self.stackId".into(),
-                PseudoParameter::StackName => "self.stackName".into(),
-                PseudoParameter::URLSuffix => "self.urlSuffix".into(),
+                PseudoParameter::StackId => "self.stack_id".into(),
+                PseudoParameter::StackName => "self.stack_name".into(),
+                PseudoParameter::URLSuffix => "self.url_suffix".into(),
```

Comment:
I know this is still in draft but just adding this question so I don't forget when I review next week. Does `self.notificationArns` also need a casing change?

Reply:
yup good catch